include/mbedtls/ssl.h

```diff
@@ -1354,8 +1354,16 @@ struct mbedtls_ssl_config
 #endif /* MBEDTLS_SSL_ASYNC_PRIVATE */
 
 #if defined(MBEDTLS_KEY_EXCHANGE_WITH_CERT_ENABLED)
-    const int *MBEDTLS_PRIVATE(sig_hashes);          /*!< allowed signature hashes           */
-#endif
+
+#if defined(MBEDTLS_SSL_PROTO_TLS1_2)
+    const int *MBEDTLS_PRIVATE(sig_hashes);             /*!< allowed signature hashes           */
+#endif /* defined(MBEDTLS_SSL_PROTO_SSL3) */
+
+#if defined(MBEDTLS_SSL_PROTO_TLS1_3_EXPERIMENTAL)
+    const int* MBEDTLS_PRIVATE(signature_algorithms);   /*!< allowed signature algorithms in TLS 1.3 */
```

Comment:
The name should indicate that this is specific to TLS 1.3 for now. Somethinglike `tls13_sig_algs`.